src/core/server/events/events.ts

```diff
@@ -106,7 +120,26 @@ export type CommentReleasedCoralEventPayload = CoralEventPayload<
 
 export const CommentReleasedCoralEvent = createCoralEvent<
   CommentReleasedCoralEventPayload
->(CoralEventType.COMMENT_RELEASED);
+>(CoralEventType.COMMENT_RELEASED, {
+  forward: [CommentEnteredCoralEvent],
+});
+
+export interface CommentReplyReleasedInput {
+  ancestorIDs: string[];
+  commentID: string;
+  storyID: string;
+}
+
+export type CommentReplyReleasedCoralEventPayload = CoralEventPayload<
+  CoralEventType.COMMENT_REPLY_RELEASED,
+  CommentReplyReleasedInput
+>;
+
+export const CommentReplyReleasedCoralEvent = createCoralEvent<
+  CommentReplyReleasedCoralEventPayload
+>(CoralEventType.COMMENT_REPLY_RELEASED, {
+  forward: [CommentEnteredCoralEvent],
+});
```

Comment:
Besides forwarding this event to the `CommentEnteredCoralEvent`, it does not appear that it's published events are handled anywhere. Maybe only calling `CommentEnteredCoralEvent` is better in this circumstance?

Reply:
Hmm yeah maybe